code/game/objects/effects/spawners/random/structure.dm

```diff
@@ -6,14 +6,14 @@
 	name = "crate spawner"
 	icon_state = "crate"
 	loot = list(
-		/obj/effect/spawner/random/structure/crate_empty = 75,
-		/obj/structure/closet/crate/trashcart/filled = 7,
-		/obj/effect/spawner/random/trash/moisture_trap = 5,
-		/obj/effect/spawner/random/trash/hobo_squat = 3,
-		/obj/structure/closet/mini_fridge = 3,
-		/obj/item/kirbyplants/random = 3,
-		/obj/effect/spawner/random/trash/mess = 3,
-		/obj/structure/closet/crate/decorations = 1,
+		/obj/effect/spawner/random/structure/crate_loot = 745,
+		/obj/structure/closet/crate/trashcart/filled = 75,
+		/obj/effect/spawner/random/trash/moisture_trap = 50,
+		/obj/effect/spawner/random/trash/hobo_squat = 30,
+		/obj/structure/closet/mini_fridge = 35,
+		/obj/item/kirbyplants/fern = 20,
+		/obj/effect/spawner/random/trash/mess = 30,
+		/obj/structure/closet/crate/decorations = 15,
```

Comment:
In discord we talked about the total weights values for items.  While there are only a few spawners that use 1000 as total weight, it's only for rare items and none of the objects in this loot list are considered rare.  

It's not a hard rule but a suggestion.  It's easier for mappers and coders to determine percents when the total weight is easy numbers like 100.  See: 

https://github.com/tgstation/tgstation/blob/master/code/game/objects/effects/spawners/random/readme.md#try-to-keep-the-total-combined-weight-of-your-loot-list-to-sane-values-aim-for-5-10-20-50-or-100

Reply:
Also move the fern to be below the mess spawner since it's a lower chance.

Reply:
Because we have so many crate spawners on the map, i believe we need to use 1000 total weight to get fine granular control over how often the rare loot site types spawn.

Especially if we want to add even rarer crate types in the future.

Reply:
1000 is a sane value. I can't imagine in what world an adult cant understand that 20/1000th is 2% for example

Reply:
@Krysonism also since the crate type has changed it'd be a good idea to update the icon state to use the secure/abandoned crate.

@AMonkeyThatCodes Ideally you would use the smallest denomination possible is what I'm trying to get at.

Reply:
1000 appears to be the smallest, sanest denomination possible to me? If it was something like 500 that would be more confusing.